be/src/olap/snapshot_manager.cpp

```diff
@@ -79,6 +79,17 @@ Status SnapshotManager::make_snapshot(const TSnapshotRequest& request, string* s
         LOG(WARNING) << "failed to get tablet. tablet=" << request.tablet_id;
         return Status::Error<TABLE_NOT_FOUND>();
     }
+    // if one be received one req with __isset.missing_version = false it means
+    // this req is sent from FE(FE doesn't know the version details and would never set this)
+    // we need to check if this tablet is one cooldown tablets return not support if so
+    if (!request.__isset.missing_version) {
+        std::shared_lock rlock(ref_tablet->get_header_lock());
+        if (ref_tablet->tablet_meta()->cooldown_meta_id().initialized()) {
```

Comment:
We MUST judge `cooldown_meta_id` in the locked area of `capture_consistent_rowsets`